src/main/java/com/atlassian/bitbucket/jenkins/internal/credentials/JenkinsToBitbucketCredentials.java

```diff
@@ -23,4 +26,30 @@
      * @param credentials, credentials
      */
     BitbucketCredentials toBitbucketCredentials(Credentials credentials);
```

Comment:
Not your code, but `credentials` here and `credentialsId` above should both be marked as `Nullable`

Reply:
Its meant to be non-nullable. the one which takes the server configuration could be nullable.